src/server/models/event.js

```diff
@@ -51,6 +51,16 @@ var EventSchema = new Schema({
     trim: true,
     required: false
   },
+  thirdPartyText: {
+    type: String,
+    default: '',
+    trim: true
+  },
+  organisedBy: {
+    type: String,
+    default: 'TESC',
+    trim: true
```

Comment:
`required: true`
If we have the default of TESC then we would expect everything to have a value.